src/libraries/System.Runtime.Caching/src/System/Runtime/Caching/HostFileChangeMonitor.Browser.cs

```diff
@@ -0,0 +1,43 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+
+namespace System.Runtime.Caching
+{
+    public sealed partial class HostFileChangeMonitor : FileChangeMonitor
+    {
+        public HostFileChangeMonitor(IList<string> filePaths)
+        {
+            throw new PlatformNotSupportedException();
+        }
+
+        public override ReadOnlyCollection<string> FilePaths
+        {
+            get
+            {
+                throw new PlatformNotSupportedException();
+            }
+        }
+
+        public override DateTimeOffset LastModified
+        {
+            get
+            {
+                throw new PlatformNotSupportedException();
+            }
+        }
+
+        public override string UniqueId
+        {
+            get
+            {
+                throw new PlatformNotSupportedException();
+            }
+        }
+
+        protected override void Dispose(bool disposing)
+        {
+            throw new PlatformNotSupportedException();
```

Comment:
`Dispose` shouldn't throw, otherwise you'll get an exception in the finalizer.

Reply:
Where is there a finalizer?

Reply:
This should be simply no-op as the type cannot be constructed